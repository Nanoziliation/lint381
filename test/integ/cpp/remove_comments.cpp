// good
// *** bad ***
/* good */

foo

/* *** bad *** */

/* You should delete this comment. */
