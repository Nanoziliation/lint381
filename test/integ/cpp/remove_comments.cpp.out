remove_comments.cpp:2:1: error: Remove triple-asterisk comments
// *** bad ***
^^^^^^^^^^^^^^
remove_comments.cpp:7:1: error: Remove triple-asterisk comments
/* *** bad *** */
^^^^^^^^^^^^^^^^^
remove_comments.cpp:9:1: error: Remove this comment
/* You should delete this comment. */
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
